src/mocksolverinstance.jl

```diff
@@ -62,8 +58,10 @@ MOI.addconstraint!(mock::MockSolverInstance, F::MOI.AbstractFunction, S::MOI.Abs
 MOI.optimize!(mock::MockSolverInstance) = (mock.solved = true)
 
 MOI.canset(mock::MockSolverInstance, ::Union{MOI.ResultCount,MOI.TerminationStatus,MOI.ObjectiveValue,MOI.PrimalStatus,MOI.DualStatus,MOI.ObjectiveSense,MOI.ObjectiveFunction,MockInstanceAttribute}) = true
-MOI.canset(mock::MockSolverInstance, ::Union{MOI.VariablePrimal,MOI.VariableName,MockVariableAttribute}, ::Type{MOI.VariableIndex}) = true
-MOI.canset(mock::MockSolverInstance, ::Union{MOI.ConstraintDual,MOI.ConstraintName,MockConstraintAttribute}, ::Type{<:MOI.ConstraintIndex}) = true
+MOI.canset(mock::MockSolverInstance, ::Union{MOI.VariablePrimal,MockVariableAttribute}, ::Type{MOI.VariableIndex}) = true
+MOI.canset(mock::MockSolverInstance, attr::MOI.AbstractVariableAttribute, idx::Type{MOI.VariableIndex}) = MOI.canset(mock.instance, attr, idx)
```

Comment:
I know I'm responsible, but the name `idx` is confusing here. We should use a different name when referring to the type.

Reply:
I replaced it with `IdxT` :)